try
{
	${1:Code}
}
catch Exception($e)
{
	${2:Code}
}
